yang-semver/draft-ietf-netmod-yang-semver.xml

```diff
@@ -217,6 +215,20 @@
 </ul>
 </li>
 </ul>
+<t>The _COMPAT modifier string is "sticky". Once a revision of a module 
+has a modifier in the revision label, then all descendants of that 
+revision will also have a modifier. The modifier can change from 
+_compatible to _non_compatible in a descendant revision, but the 
+modifier would never change from _non_compatible to _compatible 
```

Comment:
MUST NEVER?  would never seems too passive.

Reply:
updated with MUST NOT